api/integrations_ctr_reg.go

```diff
@@ -124,6 +124,12 @@ func (svc *IntegrationsService) UpdateContainerRegistry(integration ContainerReg
 	return
 }
 
+// ListContainerRegistries lists the CONT_VULN_CFG external integrations available on the Lacework Server
+func (svc *IntegrationsService) ListContainerRegistries() (response ContainerRegIntResponse, err error) {
```

Comment:
Suggestion for renaming this function as it doesn't return just the registries as a list of strings like `getContainerRegistries()` does
```suggestion
// ListContainerRegistryIntegrations lists the CONT_VULN_CFG external integrations available on the Lacework Server
func (svc *IntegrationsService) ListContainerRegistryIntegrations() (response ContainerRegIntResponse, err error) {
```